channels/lb.m3u

```diff
@@ -1,4 +1,10 @@
 #EXTM3U
+#EXTINF:-1 tvg-logo="https://i.imgur.com/Dla19Aw.png",Aghani Aghani
+https://svs.itworkscdn.net/aghanilive/aghanilive/chunklist.m3u8
+#EXTINF:-1 tvg-logo="https://i.imgur.com/N1bXAHq.png",Al Waad TV
+https://bcliveunivsecure-lh.akamaihd.net/i/AlWaad_1@772935/index_1100_av-p.m3u8?sd=10&rebase=on
+#EXTINF:-1 tvg-logo="https://i.imgur.com/lcYwUYo.png",Tele Liban
+http://cdn.catiacast.video/tl/38ad46d994b30fa6aed632fc597ca329.sdp/tracks-v1a1/mono.m3u8
```

Comment:
not working now
<img width="994" alt="Screenshot 2019-10-20 at 19 55 00" src="https://user-images.githubusercontent.com/7253922/67163110-a49c3500-f373-11e9-8ebf-80e8dcac6dbf.png">